### PR TITLE
[Sushmita][Changing the Carousel Library & Removing the extra packages]

### PR DESCRIPTION
**Pull request by Sushmita-Ghosh**
## Description

Changed the carousel library to the one we use and removed the extra libraries

## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

- Related Issue # https://digitalaidseattle-gfl-team.atlassian.net/browse/SCRUM-2
- Closes #https://digitalaidseattle-gfl-team.atlassian.net/browse/SCRUM-2

## QA Instructions, Screenshots, Recordings

We need to check that in Hospital Landing Page , the carousel is working after this change. It should display three images per slide according to the Figma Design.
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,17 +22,13 @@
     "react-dom": "^18.2.0",
     "react-map-gl": "^7.1.7",
     "react-multi-carousel": "^2.8.5",
```
[the remark this answers is not in the document]
Comment (Sushmita-Ghosh) [reply package.json:24]: Hi Jeff , While building Hospital Page Carousel, I was using the below libraries for the carousel:   
"react-slick": "^0.30.2",
"slick-carousel": "^1.8.1",
 
 I am currently using the react-responsive-carousel as it's being used HospitalCard as well. 
 
 I don't think the react-multi-carousel is being used , but I don't think if I added it. Should I remove this ?

[the remark this answers is not in the document]
Comment (jnakaso) [reply package.json:24]: yes please (and test).  thank you

[the remark this answers is not in the document]
Comment (Sushmita-Ghosh) [reply package.json]: Removed the library `react-multi-carousel` & tested

**Review by jnakaso — APPROVED** [review]
thanks for cleaning this up.  It's much appreciated.

**Pull request merged**